Documentation/architecture/net-platform-standard.md

```diff
@@ -50,24 +50,24 @@ Each platform has it own rules for how it anchors certain assemblies. In .NET Fr
 In general, class libraries which target a lower .NET Platform Standard version, like 1.0, can be loaded by the largest number of existing platforms, but will have access to a smaller set of APIs. On the other hand, class libraries which target a higher .NET Platform Standard version, like 1.3, can be loaded by a smaller number of newer platforms, but will have access to a larger, more recent set of APIs.
 
 | Target Platform Name | Alias |  |  |  |  |  | | |
-| :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Platform Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 |
-|.NET Core|netcoreapp|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|1.0|
-|.NET Framework|net|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|4.6.3|
-|||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|4.6.2|
-|||&rarr;|&rarr;|&rarr;|&rarr;|4.6.1|||
-|||&rarr;|&rarr;|&rarr;|4.6||||
-|||&rarr;|&rarr;|4.5.2|||||
-|||&rarr;|&rarr;|4.5.1|||||
-|||&rarr;|4.5||||||
-|Universal Windows Platform|uap|&rarr;|&rarr;|&rarr;|&rarr;|10.0|||
-|Windows|win|&rarr;|&rarr;|8.1|||||
-|||&rarr;|8.0||||||
-|Windows Phone|wpa|&rarr;|&rarr;|8.1|||||
-|Windows Phone Silverlight|wp|8.1|||||||
-|||8.0|||||||
-|Mono/Xamarin Platforms||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|*|
-|Mono||&rarr;|&rarr;|*||||||
+| :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
+|.NET Platform Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 | 1.7 |
+|.NET Core|netcoreapp|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|1.0|1.1|
+|.NET Framework|net|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|4.6.3|
+|||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|4.6.2|||
+|||&rarr;|&rarr;|&rarr;|&rarr;|4.6.1||||
+|||&rarr;|&rarr;|&rarr;|4.6|||||
+|||&rarr;|&rarr;|4.5.2||||||
+|||&rarr;|&rarr;|4.5.1||||||
+|||&rarr;|4.5|||||||
+|Universal Windows Platform|uap|&rarr;|&rarr;|&rarr;|&rarr;|10.0||||
+|Windows|win|&rarr;|&rarr;|8.1||||||
+|||&rarr;|8.0|||||||
+|Windows Phone|wpa|&rarr;|&rarr;|8.1||||||
+|Windows Phone Silverlight|wp|8.1||||||||
+|||8.0||||||||
+|Mono/Xamarin Platforms||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|*|
+|Mono||&rarr;|&rarr;|*|||||||
```

Comment:
I don't see the new column in the table as rendered by GitHub. I believe you also need to add a new column on the very first row of the table.